modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/simplifiedcircuit/v0.0.1.json

```diff
@@ -0,0 +1,332 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/simplifiedcircuit/v0.0.1/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.0",
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:SimplifiedCircuitShape",
+      "@type": "sh:NodeShape",
+      "targetClass": "nsg:SimplifiedCircuit",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape"
```

Comment:
For symmetry with `DetailedCircuitShape`, this should be `"{{base}}/schemas/neurosciencegraph/simulation/modelinstance/v0.1.0/shapes/ModelInstanceShape"`